Signum.Engine.Extensions/Word/WordImageReplacer.cs

```diff
@@ -82,13 +83,19 @@ private static ImageFormat ToImageFormat(ImagePartType imagePartType)
             throw new InvalidOperationException("Unexpected {0}".FormatWith(imagePartType));
         }
 
-        static Blip FindBlip(WordprocessingDocument doc, string titleOrDescription)
+        static Blip FindBlip(WordprocessingDocument doc, string titleOrDescription, bool allowMultiple = false)
         {
-            var drawing = doc.MainDocumentPart.Document.Descendants().OfType<Drawing>().SingleEx(r =>
+            var drawing = doc.MainDocumentPart.Document.Descendants().OfType<Drawing>().First(r =>
             {
                 var prop = r.Descendants<DocProperties>().SingleOrDefault();
+                var match = prop != null && (prop.Title == titleOrDescription || prop.Description == titleOrDescription);
+
+                if (allowMultiple && match)
+                {
+                    prop.Title += Guid.NewGuid().ToString();

```

Comment:
What does line does?

Reply:
It changes the DocProperty xml tag to a different one so it only gets matched once